README.md

```diff
@@ -733,6 +730,151 @@ Error Connecting to auth service ...
 }
 ```
 
+## Parse import specifications
+
+This endpoint parses one or more import specification files in the staging area into a data
+structure (close to) ready for insertion into the Narrative bulk import cell.
+
+It can parse `.tsv`, `.csv`, and Excel (`.xls` and `.xlsx`) files. Templates for the currently
+supported data types are available in the
+[templates](./import_specifications/templates) directory of this repo.
+See the `README.md` file for instructions on template usage.
+
+See the [import specification ADR document](./docs/import_specifications.ADR.md) for design
+details.
+
+**URL** : `ci.kbase.us/services/staging_service/bulk_specification`
+
+**local URL** : `localhost:3000/bulk_specification`
+
+**Method** : `GET`
+
+**Headers** : `Authorization: <Valid Auth token>`
+
+### Success Response
+
+**Code** : `200 OK`
+
+**Content example**
+
+```
+GET bulk_specification/?files=file1.<ext>[,file2.<ext>,...]
+```
+`<ext>` is one of `csv`, `tsv`, `xls`, or `xlsx`.
+
+Reponse:
+```
+{"types": {
```

Comment:
Kinda picky, but all other JSON block examples in the file look like:
```
{
    "key": {
        "other_key": "value"
    }
}
```
I'd vote for the extra return for consistency.

Reply:
fixed